### PR TITLE
Fast checking command

### PR DESCRIPTION
**Pull request by avsetsin**

### CODE REVIEW

**Review by kolyasapphire — CHANGES_REQUESTED** [review]
Also needs `packages` modifications in `pyproject.toml`, probably:
```
[
	{ include = "*.py" },
]
```
This is needed for proper packaging of all needed modules in the final package.

File: README.md [hunk 1]
```diff
@@ -37,16 +37,18 @@ lido-cli --rpc https://mainnet.provider.io/v3/XXX --max_multicall 300 --lido_add
 ```
 
 ```
---rpc                                   RPC provider for network calls.
+--rpc						RPC provider for network calls.
```
[the remark this answers is not in the document]
Comment (avsetsin) [reply README.md]: I replaced tabs in the whole block with spaces. So, now it works the same everywhere

File: README.md [hunk 2]
```diff
@@ -55,6 +57,17 @@ Example:
...
 
+### Fast Checking Network Keys
+
+A faster version of the previous command. It uses cache and excludes used keys from validating (the duplication check still runs for all loaded keys).  
+Command: `validate_network_keys_fast`
+
+Example:
+
+```
+lido-cli --rpc https://mainnet.provider.io/v3/XXX --clear_cache=False validate_network_keys_fast
```
[the remark this answers is not in the document]
Comment (avsetsin) [reply README.md]: Yes, you are right. I added the extra example

**Review by kolyasapphire — APPROVED** [review]

**Review by ujenjt — APPROVED** [review]
LGTM

**Pull request merged**